test/run-tests.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-cd "$( dirname "${BASH_SOURCE[0]}" )" && vim -Nu vimrc -c 'Vader! *' > /dev/null
+cd "$( dirname "${BASH_SOURCE[0]}" )" && /usr/local/bin/vim -Nu vimrc -c 'Vader! *' > /dev/null
```

Comment:
@cirosantilli I did do that a bit quick, just realized local testing will be impacted. I'll just overwrite the `/usr/bin/vim` file during the travis build. Simple enough.